charts/incubator/ps3netsrv/values.yaml

```
@@ -0,0 +1,20 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: shawly/ps3netsrv
+  tag: v1.9.0@sha256:c6e344b636a74e4f034a74ce766cd859071f5e46a14a0a2895ac819e66db2107
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+service:
+  main:
+    ports:
+      main:
+        port: 38008
+        targetPort: 38008
```

```suggestion
        targetPort: 38008
workload:
  main:
    podSpec:
      containers:
        main:
          env:
            USER_ID: "{{ .Values.securityContext.container.runAsUser }}"
            GROUP_ID: "{{ .Values.securityContext.container.runAsGroup }}"
          
```
As this container uses different env for PUID and PGID